compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangNodeTransformer.java

```diff
@@ -1240,6 +1242,25 @@ public BLangNode transform(AnnotationDeclarationNode annotationDeclarationNode)
         return annotationDecl;
     }
 
+    @Override
+    public BLangNode transform(AnnotAccessExpressionNode annotAccessExpressionNode) {
+        BLangAnnotAccessExpr annotAccessExpr = (BLangAnnotAccessExpr) TreeBuilder.createAnnotAccessExpressionNode();
+
+        try {
```

Comment:
lets get rid of the type cast and replace with a check on kind

Reply:
Done